Tools/scripts/build_iofirmware.py

```diff
@@ -28,3 +28,9 @@ def run_program(cmd_list):
 run_program(["./waf", "iofirmware"])
 shutil.copy('build/iomcu/bin/iofirmware_lowpolh.bin', 'Tools/IO_Firmware/iofirmware_lowpolh.bin')
 shutil.copy('build/iomcu/bin/iofirmware_highpolh.bin', 'Tools/IO_Firmware/iofirmware_highpolh.bin')
+
+run_program(["./waf", "configure", "--board", 'iomcu_f103_8MHz'])
+run_program(["./waf", "clean"])
+run_program(["./waf", "iofirmware"])
+shutil.copy('build/iomcu_f103_8MHz/bin/iofirmware_lowpolh.bin', 'Tools/IO_Firmware/iofirmware_f103_8MHz_lowpolh.bin')
```

Comment:
these file names don't match the iomcu_f103_8MHz_bl.bin in the PR

Reply:
I was following convention for the existing iofirmware_xxxxx.bin naming. Given that the io firmware build creates lowpolh and highpolh I wasn't sure the best way to handle this. open to suggestions.